src/builder-module.c

```diff
@@ -341,6 +341,8 @@ builder_module_set_property (GObject      *object,
 
     case PROP_CMAKE:
       self->cmake = g_value_get_boolean (value);
+      if (self->cmake)
+        g_printerr ("The cmake module property is deprecated, use buildsystem cmake instead.\n");
```

Comment:
Maybe mention `cmake-ninja` also?

Reply:
I dunno how important that is for the warning.

Reply:
Well it just avoids wasting everybodies time IMO:

- User sees new warning, follows warning and uses `buildsystem: cmake`
- Somebody else points out `cmake-ninja` exists now and is just better
- User updates it a second time

Reply:
Changing to cmake-ninja is not just a deprecation avoidance though, but could cause build issues. I guess its worth mentioning though.